packages/transformer/src/IModelImporter.ts

```diff
@@ -63,6 +63,11 @@ export interface IModelImportOptions {
    * @see [IModelImporter Options]($docs/learning/transformer/index.md#IModelImporter)
    */
   autoExtendProjectExtents?: boolean | { excludeOutliers: boolean };
+  /**
+   * Indicates whether the importer is derived from a transformation process.
+   * If `true`, the importer originates from a transformation.
```

Comment:
You say the importer originates from a transformation, but I think that could be confusing because we could construct an instance of the importer and pass it to the transformer and `fromTransformation` would still be true right?

Reply:
Yeah, I can see how that will be confusing.

I was looking at it somehow wrongly, from the point of view where inside the transformation code in the constructor, the importer instance is created if it is not passed into the transformer but we could also pass it into the transformer